### PR TITLE
Fix farmedit modal

### PR DESCRIPTION
**Pull request by samaradel**
### Description

Fix delete request 

### Changes

- Display IP in delete modal
- Fix delete request

### Related Issues

[#2701](https://github.com/threefoldtech/js-sdk/issues/2701)

### Checklist

- [ ] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: jumpscale/packages/vdc/package.toml [hunk 1]
```diff
@@ -22,8 +22,6 @@ port = 5562
 
 [[servers]]
 name = "vdc_proxy"
-domain = "vdc.testnet.grid.tf"
-letsencryptemail = "motaweam@incubaid.com"
```
Comment (waleedhammam) [inline jumpscale/packages/vdc/package.toml:26]: need to bring back, we can leave values empty

File: jumpscale/sals/vdc/wallet.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def stellar_wallet(self):
 class VDCWalletStoredFactory(StoredFactory):
     def new(self, *args, **kwargs):
         instance = super().new(*args, **kwargs)
-        instance._init_wallet()
+        #instance._init_wallet()
```
Comment (waleedhammam) [inline jumpscale/sals/vdc/wallet.py:28]: need to bring back

File: jumpscale/packages/threebot_deployer/package.toml [hunk 3]
```diff
@@ -29,8 +29,6 @@ port = 5561
 
 [[servers]]
 name = "threebot_deployer_root_proxy"
-domain = "deploy3bot.testnet.grid.tf"
-letsencryptemail = "motaweam@incubaid.com"
```
Comment (waleedhammam) [inline jumpscale/packages/threebot_deployer/package.toml:33]: need to bring back, we can leave values empty

File: jumpscale/packages/farmmanagement/frontend/views/farmedit/farmedit.js [hunk 4]
```diff
@@ -88,8 +88,9 @@ module.exports = new Promise(async (resolve, reject) => {
             deleteIpAddressFarmer (ipaddress) {
                 const del = {
                     farm_id: this.farm.id,
-                    ipaddress: ipaddress.ipaddress
+                    ipaddress: ipaddress.address
                 }
+                console.log(ipaddress)
```
Comment (waleedhammam) [inline jumpscale/packages/farmmanagement/frontend/views/farmedit/farmedit.js:93]:
```suggestion
```

**Review by waleedhammam — CHANGES_REQUESTED** [review]

**Review by waleedhammam — APPROVED** [review]

**Review by rkhamis — APPROVED** [review]

**Pull request merged**